samples/msteams-application-qbot/Deployment/README.md

```diff
@@ -50,6 +50,11 @@ The script will write deployment logs to `deployment.log` file in the `scripts`
 
 Make sure that you have upgraded to Powershell [5.1.22000.653 or later](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-windows?view=powershell-7.2). Run `$PSVersionTable.PSVersion` to check the installed version.
 
+### Max students in a Team
+The solution is tested for upto 1k users in a Team. If your tenant has more than 1k users in a Team, we suggest you to make a few changes to the app logic.
```

Comment:
Let's spell out 1,000 and lowercase Team ("up to 1,000 users in a team")

Reply:
Also see the capitalized "Team" in the paragraph below

Reply:
Fixed.